src/main/java/org/moltimate/moltimatebackend/alignment/Alignment.java

```diff
@@ -4,13 +4,20 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.util.List;
+
 /**
  * Represents the results of performing a single alignment between two active sites.
  */
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class Alignment {
-
+    String protein;
+    String motif;
```

Comment:
What value does `motif` store? It might be ambiguous because we also have a class called Motif

Reply:
Changed to motifName and proteinName